MVP_rev1.py

```diff
@@ -0,0 +1,1168 @@
+'''''
+Interactive Programming: Visualizing Music
+
+author@ Maggie Rosner & Aurora Bunten
+
+03/01/17
+'''''
+
+import pygame
+import pyaudio
+import wave
+import sys
+
+class Shape(object):
+    '''
+    the shaps class creates objects that are defined by the note they represent
+    '''
+    def __init__(self, note):
+        self.note = note
+        self.disp = False
+
+
+class Controller(object):
+    '''
+    The controller class calls a specific shape object based on the key pressed.
+    the shape is associated with a specific note which will change the shape_disp
+    to True and play the wave file associated with the note.
+    '''
+    def __init__(self, model):
+        self.model = model
+        self.octave = 4     #sets the default octave to the 4th (middle C)
+    def input_event(self,event):
+        is_audio_event = False  #sets the initial condition for an event (shape display) to False
+        '''
+        the conditions below set the octave based on certain number keys that are pressed
+        '''
+        if event.type == pygame.KEYDOWN: #if a key is pressed
+            if event.key == pygame.K_5: #if the number 5 key is pressed
+                self.octave = 5 #play notes in the 5th octave
+            elif event.key == pygame.K_4:   #if the number 4 key is pressed
+                self.octave = 4 #play notes in the 4th octave
+            elif event.key == pygame.K_3:   #if the number 3 key is pressed
+                self.octave = 3 #play notes in the 3rd octave
+
+            if self.octave == 4:
+                '''
+                Notes A - G can be played with keys A - J on
+                a standard keyboard. A# - G# can be played with keys Q - U. Ab - Gb
+                can be played with keys Z - M. If you simply push down the key assigned
+                to a note it will play an 8th note in that tone.
+                If you push down shift and the key assigned the note played will be a
+                half note in that octave. Each differet note has a different wave
+                file assigned to it that will play when the correct key/keys are pressed.
+                '''
```

Comment:
Same thing here. There is no need to explain the entire mechanics of how your program works. Comments should be something that readers of your code can use to facilitate their understanding of code. Nothing more than that. More appropriate place to put explanation like this would be in `README.md` file.